app/mailers/application_mailer.rb

```diff
@@ -1,4 +1,20 @@
+# frozen_string_literal: true
+
 class ApplicationMailer < ActionMailer::Base
-  default from: 'from@example.com'
-  layout 'mailer'
```

Comment:
I think we still need default using env vars
and make sure we have a layout

Reply:
1. We get sender_name and sender_email from the ENV.